contracts/MerkleDelegation.sol

```diff
@@ -35,34 +31,26 @@ contract MerkleDelegation is Ownable, Pausable {
     constructor(address _governanceToken) Ownable() Pausable() {
         require(
             _governanceToken != address(0),
-            "DR: token addr must be non-zero"
+            "MD: token addr must be non-zero"
         );
         governanceToken = _governanceToken;
     }
 
-    //TODO: also the owner/admin can update the governance token address that is hashed with all leafs.
-    // function updateGovernanceToken(address newGovernanceToken) { governanceToken = newGovernancetoken; }
-
     function setDelegateTrie(address delegator, bytes32 trieRoot)
         external
         whenNotPaused
     {
-        require(delegator != address(0), "DR: delegator must be non-zero");
-        require(trieRoot != bytes32(0), "DR: trieRoot must be non-zero");
-        require(msg.sender == delegator, "DR: delegator must be msg.sender");
+        require(delegator != address(0), "MD: delegator must be non-zero");
+        require(trieRoot != bytes32(0), "MD: trieRoot must be non-zero");
+        require(msg.sender == delegator, "MD: delegator must be msg.sender");
```

Comment:
consider using _msgSender() if we are importing this anyway? (not sure if we are)

Reply:
changed to _msgSender().